cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/engine/sql/request/SQLWriteResult.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.etl.api.engine.sql.request;
+
+import io.cdap.cdap.api.annotation.Beta;
+import io.cdap.cdap.etl.api.engine.sql.SQLEngineOutput;
+
+import java.io.Serializable;
+
+/**
+ * A request to perform write operation
+ */
+@Beta
+public class SQLWriteResult implements Serializable {
+  private final String datasetName;
+  private final SQLWriteOperationResult result;
+  private final long numRecords;
+
+  public SQLWriteResult(String datasetName, SQLWriteOperationResult result, long numRecords) {
```

Comment:
Add javadocs to constructor / public methods

Reply:
Added